cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -91,6 +90,17 @@ def counts_to_results(
         # Appends all the keys onto 'samples' list number-of-counts-in-the-key times
         # If collections.Counter({"01": 48, "11": 52}), [0, 1] is appended to 'samples` 48 times and
         # [1, 1] is appended to 'samples' 52 times
+        if int(counts_of_key) != counts_of_key:
+            print(
+                warnings.warn(
+                    "The raw counts are fractional due to measurement error mitigation; please use "
+                    "service.get_counts to see raw results.",
+                    UserWarning,
+                    stacklevel=2,
+                )
+            )
+
+        counts_of_key = int(counts_of_key)
```

Comment:
I think that this maybe should be `round` instead of int, if we're expecting to something get something like `{"0": 7.1, "1": 2.9}`. In either case, there's the potential of getting values that don't add up to the original total, but I think `round` would generally get us closer.

Reply:
Good point -- done.